### ISSUE TITLE
Already trained model.

### ISSUE THREAD

**Issue by berkantay** [nrupatunga/goturn-pytorch#9]
Hello @nrupatunga i am planning to use goturn tracker with yolov5. I have my trained weights and detection working well. Now I want to integrate goturn tracker to my code. I have ```my_custom_model.pt```. Can I use this for tracking?

**Comment by nrupatunga** [comment]
yes, so if I understand correctly, your object detection model gives an initial bounding box for the tracking to continue, 

so you can refer to the following test script(might need some modifications) and can set your initial bounding box. 
https://github.com/nrupatunga/goturn-pytorch/blob/bb3b3b418ac361aa5782b0a7569cc2191b1c30fc/src/scripts/demo_folder.py#L213 


How to test: This will guide as to how to test for new video sample
https://github.com/nrupatunga/goturn-pytorch#testing

**Comment by berkantay** [comment]
Yes exactly, my network gives the bounding box coordinates. I am aiming to use those coordinates to initiate goturn tracker.

**Comment by berkantay** [comment]
@nrupatunga I have one more question. Yolov5 framework that I use for detection uses newer version of torch, pillow etc. would it matter if I mind to use those versions for your code?

**Comment by nrupatunga** [comment]
Since I'm using Pytorch-Lightning, there are few changes to it in the new version. 
Let me know if you face any issues, would look into it.

**Comment by berkantay** [comment]
I got segmentation fault when I press """s""" for tracking.
